### ISSUE TITLE
'child_options' doesn't appear in the help

### ISSUE THREAD

**Issue by humitos** [PyAr/fades#93]
```
$ fades -h
usage: PROG [-h] [-V] [-v] [-q] [-d DEPENDENCY] [-r REQUIREMENT] [-p PYTHON]
            [child_program] ...

positional arguments:
  child_program
  child_options

optional arguments:
  -h, --help            show this help message and exit
```

I suppose that it should say `[child_options]` before `...` and `[child_program]`... Don't you think?

```
$ fades -V
Running 'fades' version 3
    Python: sys.version_info(major=3, minor=4, micro=0, releaselevel='final', serial=0)
    System: linux
```

**Comment by facundobatista** [comment]
Fixed by 90c3b624d708163f8637ec371aa725c2235cb45e

**Comment by gilgamezh** [comment]
ack!
